scripts/datetimes.py

```diff
@@ -39,6 +68,40 @@ def format_rfc_3339_nz_midnight_datetime_string(datetime_object: datetime) -> st
     return format_rfc_3339_datetime_string(datetime_utc)
 
 
+def convert_utc_to_nz(datetime_object: datetime) -> datetime:
+    """Convert a UTC datetime to New Zealand time zone (Pacific/Auckland).
+
+    >>> dt_utc = datetime(2001, 2, 2, 11, 0, 0, tzinfo=timezone.utc)
+    >>> convert_utc_to_nz(dt_utc)
+    datetime.datetime(2001, 2, 3, 0, 0, tzinfo=tzfile('...Pacific/Auckland'))
+    >>> convert_utc_to_nz(datetime(2001, 2, 3, 4, 5, 6))
+    Traceback (most recent call last):
+    ...
+    scripts.datetimes.NaiveDatetimeError: Can't convert naive datetime, timezone aware datetime object needed
+    >>> dt_nz = datetime(2001, 2, 3, 4, 5, 6, tzinfo=tz.gettz("Pacific/Auckland"))
+    >>> convert_utc_to_nz(dt_nz)
+    Traceback (most recent call last):
+    ...
+    scripts.datetimes.NotUTCError: Input datetime must be in UTC
+    """
+    if datetime_object.tzinfo is None:
+        raise NaiveDatetimeError()
+    tzinfo = datetime_object.tzinfo
+    if tzinfo != timezone.utc and str(tzinfo) not in (str(timezone.utc), str(tz.gettz("UTC"))):
```

Comment:
```suggestion
    if tzinfo != timezone.utc:
```
Would this be sufficient?

Reply:
In this case it would be sufficient, thanks :+1: